app/models/like.rb

```diff
@@ -0,0 +1,11 @@
+class Like < ApplicationRecord
+  belongs_to :post, :user
+
+  after_save :update_like_counter
+
+  private
+
+  def update_likes_counter
+    post.likes_counter += 1
```

Comment:
- [ ] <b>OPTIONAL:</b> This the proper way to increment. `post.increment!(:likes_counter)`

Reply:
This change has been done well